charts/backingservices/charts/srs/README.md

```diff
@@ -18,8 +18,7 @@ The service deployment provisions runtime service pods along with a dependency o
 | Pega Infinity version | SRS version | Elasticsearch version | Description                                                                                                                                                                                                                                                                                                           |
 |-----------------------|-------------|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | < 8.6                 | NA          | NA                    | SRS can be used with Pega Infinity 8.6 and later                                                                                                                                                                                                                                                                      |
-| \>= 8.6 & \<= 8.8.0              | \>= 1.17.10 & \< 1.20.7  | 7.10.2 or 7.16.3      | Pega Infinity 8.6 through 8.8.0 supports using a Pega-provided platform-services/search-n-reporting-service Docker image. While all SRS Docker image versions starting at 1.12.0 are certified against Elasticsearch versions 7.10.2 and 7.16.3, Pega recommends using the latest available SRS image - 1.20.7 and later.                                                                                                                                                                                                                                                                     |
-| \>= 8.6              | \>= 1.20.7  | 7.10.2 or 7.16.3 or 7.17.9     | Pega Infinity 8.6 and later supports using Pega-provided platform-services/search-n-reporting-service Docker Image version 1.20.7 or later with Elasticsearch version, 7.17.9. Pega Infinity 8.8.1 and later requires Docker image 1.20.7 and later. Docker image version 1.20.7 is backward-compatible with Elasticsearch version 7.10.2 and 7.16.3.|
+| \>= 8.6              | 1.20.7  | 7.10.2 or 7.16.3 or 7.17.9      | While SRS Docker images are certified against Elasticsearch versions 7.10.2, 7.16.3 and 7.17.9, Pega recommends using Elasticsearch version 7.17.9. To stay current with Pega releases, use the latest available SRS image 1.20.7.
```

Comment:
this looks kind of odd>7.10.2 or 7.16.3 or 7.17.9 
Can you make it>>
7.10.2, 7.16.3, and 7.17.9 

The description looks good. ;-)

Reply:
Sure!! Thanks for your inputs...

Reply:
Done.